app/views/taxonomies/show.html.erb

```diff
@@ -1,6 +1,12 @@
 <%= display_header title: '', breadcrumbs: [:taxons, @taxonomy.tree.first.title] do %>
 <% end %>
 
+<%=
```

Comment:
If you put this link inside the header block above, it will float right without the need of extra CSS I think. Have a look into https://github.com/alphagov/content-tagger/blob/master/app/views/taxons/index.html.erb#L1-L12 for an example.

It would be nice to have an icon too 😸